contracts/Libraries/ERC20.sol

```diff
@@ -2,7 +2,7 @@ pragma solidity ^0.8.4;
 
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 
-contract LibERC20 {
+contract ERC20 {
```

Comment:
We decided to prefix all libs with Lib, otherwise this causes compilation conflicts with existing interfaces that you'd have to constantly rename